installing/disconnected_install/about-installing-oc-mirror-v2.adoc

```diff
@@ -50,4 +50,36 @@ If you do not have an existing solution for a container image registry, {product
 
 * Every machine in the provisioned clusters must have access to the mirror registry. If the registry is unreachable, tasks like installation, updating, or routine operations such as workload relocation may fail. Therefore, mirror registries must be operated in a highly available manner, ensuring their availability aligns with the production availability of your {product-title} clusters.
 
-* You have set the umask parameter to `0022` on the operating system that uses oc-mirror.
\ No newline at end of file
+* You have set the umask parameter to `0022` on the operating system that uses oc-mirror.
```

Comment:
why did we have this here again ?

Reply:
It was implemented through another Jira ticket. If you check the google docs we were working on, this is added there for all the prerequisites.